src/InboundServer/api.yaml

```diff
@@ -2643,6 +2738,26 @@ components:
         see API Signature document.
       schema:
         type: string
+    traceparent:
+      name: traceparent
+      in: header
+      required: false
+      description: >-
+        The traceparent header field is a non-HTTP standard field used by the API
+        for passing along the parent traceId for children traces to use to create child
+        spans.
+      schema:
+        type: string
+    tracestate:
+      name: tracestate
+      in: header
+      required: false
+      description: >-
+        The tracestate header field is a non-HTTP standard field used by the API
```

Comment:
Please refer to the following --> https://mojaloop.io/documentation/mojaloop-technical-overview/event-framework/#411-wc3-http-headers

Also, please grab the definition from the official WC3 spec and provide a link to where people can read more information on the header.